applications/capability_integration.md

```diff
@@ -0,0 +1,63 @@
+# Integration Requirements for Argo Workflows on EOxHub
+
+This document outlines the minimum requirements for integrating a new process into the Argo Workflow system on EOxHub. This includes input definitions, expected Docker image standards, and output formats.
+
+## Input Requirements
+
+If your service should be integrated into the eodash processing widget docker input fields must correspond to our supported options. Beside this we expect to get information regarding:
+
+- Description of expected input data - does the service expect data already present or data are downloaded as part of the dockerized algorithm?
+- Description of expected argument and parameters
+- Description of expected output - see Output requirements section
+
+
+### Arguments and parameters
+
+eodash processing widget support multiple ways how to pass input. 
+- Area/location - process can take as input drawn point or polygon from the eodash user interface. For this integration, input field must accept eather coorinates directly or geoJSON as a string. File input is not accepted. 
+- Date - standard date formats are supported. eodash also supports start and end time to create range.
+- Numeric fields - integer or float values
+- Text fields
+- Dropdowns with limited options
+  
+  
+All fields can have default value
+
+
+## Docker Image Requirements
+
+Your process must be encapsulated in a Docker image without the strict limitations. It is helpful if the image is "slim" - only with required dependencies installed.
```

Comment:
Your process must be encapsulated in a Docker image. There are no strict limitations of what can be done, but there are good practices that help integration:

* image is "slim" - only with required dependencies installed
* image is tagged with version based on [semantic versioning](https://semver.org/)
* algorithm logs to standard out (stdout) helping debug potential issues